browsers/opera_android.json

```diff
@@ -18,13 +18,14 @@
           "engine_version": "2.7"
         },
         "11.1": {
-          "release_date": "2011-06-30",
+          "release_date": "2011-04-11",
```

Comment:
The linked release notes are dated June 30 2011; your date appears to be for the desktop release. I think this change can be reverted

Reply:
```suggestion
          "release_date": "2011-06-30",
```